dozer/cogs/ftc.py

```diff
@@ -75,19 +76,20 @@ async def reqjson(self, endpoint, season=None, on_400=None, on_other=None):
     @staticmethod
     def get_season():
         """Fetches the current season, based on typical kickoff date."""
-        today = datetime.datetime.today()
+        today = datetime.today()
         year = today.year
         # ftc kickoff is always the 2nd saturday of september
```

Comment:
This comment is misleading as FTC kickoff isn't *always* the 2nd Saturday of December; for example, kickoff for Freight Frenzy was 2021-09-18, which is the 3rd Saturday of December. This is a reasonable enough heuristic, but the comment should probably say typically. In theory, it’s possible to scrape e.g. <https://www.firstinspires.org/resource-library/ftc/game-and-season-info>, but that is not worth it for this rare edge-case.

Reply:
I might recommend using the official FTC API instead: https://ftc-events.firstinspires.org/api-docs/index.html#tag/General/paths/~1v2.0/get

Reply:
Will note that's not my code, only fixed the datetime section. Also not to be picky, but to my knowledge FTC kickoff is generally in September, not December.

Reply:
hi i wrote this code and this predates ftc-events existing so what i _would_ do is hit the ftc-api if the current day is in september but otherwise guess it based on month (to minimize api calls)

Reply:
If/when I have time I'll probably make a new PR for the kickoff case (it's still a few months until this year's championship so no point in fixing the command *yet*